perception/image_projection_based_fusion/CMakeLists.txt

```
@@ -1,42 +1,179 @@
 cmake_minimum_required(VERSION 3.14)
 project(image_projection_based_fusion)
+add_compile_options(-Wall -Wno-unknown-pragmas)
```

```suggestion
add_compile_options(-Wno-unknown-pragmas)
```
It's in autoware_cmake.

---

deleted.